doc/Overview/Reactive/overview.md

```diff
@@ -0,0 +1,66 @@
+---
+uid: Overview.Reactive.Feed
+---
+
+# MVUx Overview
+
+## What is MVU
+
+MVU stands for **M**odel-**V**iew-**U**pdate.  
+
+It utilizes a flow where the *model* is rendered on the *view* side, and once changed by request from user, gets *update*d.
```

Comment:
Why *updated*  is in italic? I would keep this for feed concept like *model*  and *view* like you did (and use italic every times we are referring to those concepts)